beetsplug/bandcamp/metaguru.py

```diff
@@ -95,10 +95,12 @@ def comments(self) -> Optional[str]:
 
     @cached_property
     def all_media_comments(self) -> str:
-        return "\n".join([
-            *[m.description for m in self.media_formats],
-            self.comments or "",
-        ])
+        return "\n".join(
```

Comment:
I should do better with documenting this, but this repository is formatted with `ruff format --preview`.

Reply:
Ah I see, had black still installed and overlooked that one. However, now that I switched to ruff it still formats the same way. Do you have any custom config that is not included in the repo or that I overlooked?

Reply:
Try `ruff format --preview`

Reply:
Okay I used the vscode plugin and it does not seem to work, cli I worked however; updated PR.